README.md

```diff
@@ -55,21 +55,21 @@ The build system will attempt to detect the local architecture and automatically
 All commands `build`, `tests`, and `up` will use the locally detected platform and use a version tag based on the state of the local git repository.
 However, you can also specify a custom platform or version with the `--platform` and `--version` parameters, example: `doit build --arch=arm64 --version=my-version`.
 
-You can specify target image variants to build with `--target`, example: `doit build --target base --target lab`.
```

Comment:
To simplify the code, you can now only provide a single `--target` argument (and we now automatically validate that you choose one of the four valid images. If no target is specified, then all images are built by default.